project/plugins.sbt

```diff
@@ -1,12 +1,11 @@
 resolvers ++= Seq(
-  "jgit-repo" at "http://download.eclipse.org/jgit/maven",
-  "sonatype-releases"  at "http://oss.sonatype.org/content/repositories/releases"
+  "jgit-repo" at "http://download.eclipse.org/jgit/maven"
 )
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.5.2")
-
-addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.6")
-
-addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.7.0")
-
-addSbtPlugin("com.typesafe.sbt" % "sbt-scalariform" % "1.3.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release"     % "1.0.0")
```

Comment:
Could we use scalriform? the old one is closer to how we format most of our scala I suppose